pkg/cmd/cli/backup/create.go

```diff
@@ -191,7 +186,7 @@ func (o *CreateOptions) Validate(c *cobra.Command, args []string, f client.Facto
 
 	if o.StorageLocation != "" {
 		location := &velerov1api.BackupStorageLocation{}
-		if err := client.Get(context.Background(), kbclient.ObjectKey{
+		if err := o.client.Get(context.TODO(), kbclient.ObjectKey{
```

Comment:
Could you give some more information about why use the TODO context instead of background?
Looks like Golang use the background context as the root context.

Reply:
I just keep the code as the under code .
```
for _, loc := range o.SnapshotLocations {
		snapshotLocation := new(velerov1api.VolumeSnapshotLocation)
		if err := o.client.Get(context.TODO(), kbclient.ObjectKey{Namespace: f.Namespace(), Name: loc}, snapshotLocation); err != nil {
			return err
		}
	}
```

Reply:
Would you suggest me to change the both 'context.TODO()'-->'context. Background()'.?

Reply:
Yes. Using the background context is better.

Reply:
Ok , i will separation a commit for the replace works.
I will replace all codes about that , not include the generated code.